test/unit/spec/EnrollSms_spec.js

```diff
@@ -274,6 +274,39 @@ function (Okta, OktaAuth, LoginUtil, Util, AuthContainer, Form, Beacon, Expect,
             expect(test.form.hasErrors()).toBe(true);
           });
       });
+
+      itp('on WARNING_TIMEOUT, after send/re-send, shows warning message for SMS after 30s idle', function () {  
```

Comment:
text should be just `shows warning message to click "Re-send" after 30s`

Reply:
fixed